arango/collection.py

```diff
@@ -1812,6 +1813,11 @@ def insert_many(
             index caches if document insertions affect the edge index or
             cache-enabled persistent indexes.
         :type refill_index_caches: bool | None
+        param version_attribute: support for simple external versioning to
+            document operations.
+        :type version_attribute: str
+        :return: Document metadata (e.g. document key, revision) or True if
+            parameter **silent** was set to True.
```

Comment:
This duplicates the previous `:return:` docstring, which was already there. In order to keep a uniform `:return:` string among these functions, we should remove this and leave the already existing one there.